tasks.ps1

```diff
@@ -35,9 +32,16 @@ Set-Alias -Name ce -Value CommandAliasFunction -Scope script
 
 [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUserDeclaredVarsMoreThanAssignments', '', Scope = 'Function')]
 $variantApiDeployYamlPath = [System.IO.Path]::GetFullPath((Join-Path ${RepositoryRoot} ".variant/deploy/"))
-$deployYamlsFound = Get-ChildItem -Path $variantApiDeployYamlPath -Filter "*.yaml" -Recurse
-if ($deployYamlsFound.Count -gt 0)
-{
+$testPath = Test-Path -Path $variantApiDeployYamlPath
+Write-Information "Deploy folder exists in .variant directory: $testPath"
+if ($testPath -eq $true) {
+  $deployYamlsFound = Get-ChildItem -Path $variantApiDeployYamlPath -Filter "*.yaml" -Recurse
+}
+else {
+  throw "Deploy folder does not exists in .variant directory"
```

Comment:
Can we make the error message bigger/more noticeable? Something like this:
https://github.com/variant-inc/actions-python/blob/45596e2d1fa6d9e0b7434d1b18d19631edf8f161/scripts/coverage_scan.sh#L11

Reply:
Test case working, just want to make it a bit more noticeable in the logs:
https://github.com/variant-inc/dx-demo-cron/runs/5816320669?check_suite_focus=true

Reply:
Added more noticeable error message.